odk1-config/s3_website.yml

```diff
@@ -38,4 +38,5 @@ redirects:
   briefcase-forms/index.html: briefcase-using
   briefcase-vs-aggregate/index.html: briefcase-and-aggregate
   form-widgets/index.html: form-question-types
-  form-interaction/index.html: form-logic
\ No newline at end of file
+  form-interaction/index.html: form-logic
+  odk-build/index.html: /build-intro
```

Comment:
I can't test this until it goes live, but 
(based on the existing precedent of the preceding lines)
I think this should be: